CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 - **Azure Queue:** Don't call Azure queue GetProperties API unnecessarily ([#2613](https://github.com/kedacore/keda/pull/2613))
 - **Datadog Scaler:** Validate query to contain `{` to prevent panic on invalid query ([#2625](https://github.com/kedacore/keda/issues/2625))
 - **Kafka Scaler** Make "disable" a valid value for tls auth parameter ([#2608](https://github.com/kedacore/keda/issues/2608))
+- **Azure Pipelines:** Added the ability to support ADO demands by referencing a master template ([#2707](https://github.com/kedacore/keda/pull/2707))
```

Comment:
Please sort alphabetically

Reply:
Just returned from vacation so shall look into that this week